libndt.hpp

```diff
@@ -1578,8 +1635,7 @@ bool Client::run_download() noexcept {
     // address are attempted simultaneously, the client MUST serialize them".
     Err err = netx_maybews_dial(  //
         settings_.hostname, port,
-        ws_f_connection | ws_f_upgrade | ws_f_sec_ws_accept
-          | ws_f_sec_ws_protocol, ws_proto_s2c, "/ndt_protocol",
+        ws_f_connection | ws_f_upgrade | ws_f_sec_ws_accept | ws_f_sec_ws_protocol, ws_proto_s2c, "/ndt_protocol",
```

Comment:
This is horrible, as a hack add `//` after `ws_f_sec_ws_accept` to avoid the very long line, maybe?